lib/pagination_view.dart

```diff
@@ -112,13 +128,22 @@ class PaginationViewState<T> extends State<PaginationView<T>> {
     );
   }
 
+  _buildSingleWidgetView(Widget onError) {
+    return Stack(
+      children: <Widget>[
+        onError,
+        ListView()
+      ],
+    );
+  }
+
```

Comment:
What happens when you wrap `onError` with `SingleChildScrollView` like

```dart
  _buildSingleWidgetView(Widget onError) {
    return SingleChildScrollView(
      child: onError,  // this can have a generic name as well
    );
  }
```

Reply:
when we use `SingleChildScrollView` the widget doesnt know the parent's height (unbound height). so, maybe it will not center of the screen